app/components/project-category-item.js

```diff
@@ -1,13 +1,59 @@
 import Ember from 'ember';
 
+/**
+  `project-category-item` represents a category with a project. When selected
+  the user is taken to a list of projects using this category.
+
+  ## Default usage
+  ```Handlebars
+  {{project-category-item category=category}}
+  ```
+
+  @class project-category-item
+  @module Components
+  @extends Ember.Component
+ */
 export default Ember.Component.extend({
   tagName: ['li'],
 
+  /**
+    Returns true if 'userCategory' is not empty
+
+    @property selected
+    @type Boolean
+   */
   selected: Ember.computed.notEmpty('userCategory'),
+
+  /**
+    A service that returns all userCategories.
+
+    @property userCategories
+    @type Ember.Service
+   */
   userCategories: Ember.inject.service(),
+
+  /**
+    The current user.
+
+    @property user
+    @type Ember.Object
```

Comment:
I don't have an opinion myself, really. Both works.

However, looking at how emberjs itself does it, it does look like they go with the `Ember.` prefix for most stuff. I can't find an instance of `@type Ember.Object`, though. I did find instances of `@extends Ember.Object`.

I also found instances of both `Service` and `Ember.Service` when it's clearly the same type, so who knows.

I would like to, however, note that in this case, if we want to be specific with our type, this is technically a `DS.Model`, in this case, no?

Maybe we should go with `@class DS.Model` instead of `@type Object` to be clearer?